docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -171,6 +171,7 @@ For hobby developers who want to get acquainted with certification tools/process
 
 TH runs on Ubuntu 22.04 Server LTS. The official installation method uses a Raspberry Pi (<<th-image-installation-on-raspberry-pi>>), but there's an alternative method used in the tool's development that uses a virtual machine instead (<<th-installation-without-a-raspberry-pi>>). Keep in mind that thread networking is not officially supported in VM installations at the moment.
```

Comment:
I know this is not related to your change, but I just noticed that this link is broken.

```suggestion
TH runs on Ubuntu 22.04 Server LTS. The official installation method uses a Raspberry Pi (<<th-installation-on-raspberry-pi>>), but there's an alternative method used in the tool's development that uses a virtual machine instead (<<th-installation-without-a-raspberry-pi>>). Keep in mind that thread networking is not officially supported in VM installations at the moment.
```

OR

```suggestion
TH runs on Ubuntu 22.04 Server LTS. The official installation method uses a Raspberry Pi (<<fresh_install>>), but there's an alternative method used in the tool's development that uses a virtual machine instead (<<th-installation-without-a-raspberry-pi>>). Keep in mind that thread networking is not officially supported in VM installations at the moment.
```